cmd/config.go

```diff
@@ -47,13 +47,16 @@ var configurationSetCmd = &cobra.Command{
 	Long: `Set an individual value in the Odo configuration file 
 Available Parameters:
 UpdateNotification - Controls if an update notification is shown or not (true or false)
-Timeout            - Timeout (in seconds) for openshift server connection check`,
+Timeout            - Timeout (in seconds) for OpenShift server connection check`,
 	Example: `
   # Set UpdateNotification off
   odo utils config set UpdateNotification false
 
   # Set OpenShift server connection check to 20 seconds
   odo utils config set timeout 20
+
+  # Set 0 to don't timeout requests for OpenShift server check
```

Comment:
Weird wording..

Do we really need this example? The examples above seem like enough.

Reply:
thought it will be nice to mention about 0 and its special behavior in help